extensions/qrcode-generator/src/index.tsx

```diff
@@ -1,17 +1,18 @@
-import { Action, ActionPanel, Form, getPreferenceValues, open, showToast, Toast } from "@raycast/api";
+import { Action, ActionPanel, Form, getPreferenceValues, showToast, Toast } from "@raycast/api";
 import QRCode from "qrcode";
 import { useState } from "react";
 import { generateQRCode, getQRCodePath, QRCodeView } from "./utils";
 import { FormValidation, useForm } from "@raycast/utils";
-
+import { randomBytes } from "crypto";
+var open = require('mac-open');
```

Comment:
```suggestion
import open from "mac-open";
```

(can be remove if using `showInFinder`)